ios/FluentUI/TextField/FluentTextField.swift

```diff
@@ -0,0 +1,307 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import Combine
+
+@objc(MSFTextField)
+public final class FluentTextField: UIView, UITextFieldDelegate, TokenizedControlInternal {
```

Comment:
ooc - it seems most of our controls are open, why is the textfield final?

Reply:
Seemed like it would be easier to release it as final, and then open it up for subclassing later if that was really needed.